vendor/github.com/syndtr/gocapability/LICENSE

```diff
@@ -0,0 +1,24 @@
+Copyright 2013 Suryandaru Triandana <syndtr@gmail.com>
```

Comment:
What kind of this license is it?
It is some kind of different from some usual others.